users/config.rst

```diff
@@ -228,6 +290,8 @@ Folder Element
         <disableFsync>false</disableFsync>
         <blockPullOrder>standard</blockPullOrder>
         <copyRangeMethod>standard</copyRangeMethod>
+        <caseSensitiveFS>false</caseSensitiveFS>
+        <junctionsAsDirs>true</junctionsAsDirs>
```

Comment:
```suggestion
        <junctionsAsDirs>false</junctionsAsDirs>
```

Sticking to the convention of using default values here, shouldn't this say `false`? The default for `junctionsAsDirs` is `false`.

Reply:
Done.  I have no clue how it ended up there, because I just migrated the existing config.xml content. from the example.